blockprod/src/rpc.rs

```diff
@@ -22,54 +22,59 @@ use common::{
 
 use serialization::{hex::HexDecode, hex::HexEncode};
 
-use crate::BlockProductionError;
+use crate::{detail::JobKey, BlockProductionError};
 use subsystem::subsystem::CallError;
 
 #[rpc::rpc(server, namespace = "blockprod")]
 trait BlockProductionRpc {
     /// Stop block production
-    #[method(name = "stop")]
-    async fn stop(&self) -> rpc::Result<()>;
+    #[method(name = "stop_all")]
+    async fn stop_all(&self) -> rpc::Result<()>;
 
-    /// Start block production on the next chance (when new tip is available)
-    #[method(name = "start")]
-    async fn start(&self) -> rpc::Result<()>;
+    // Stop a specific job
+    #[method(name = "stop_job")]
+    async fn stop_job(&self, job_id: JobKey) -> rpc::Result<()>;
```

Comment:
Should `JobKey` be a `String` that we parse on input instead?

Reply:
It can be some serialized hex. That's fine. Eventually it'll contain a key too.